apps/service_providers/llm_service/adapters.py

```diff
@@ -264,16 +243,12 @@ def get_file_type_info(self, attachments: list) -> list:
     def get_attachments(self, attachment_type: list[str]):
         return self.session.chat.attachments.filter(tool_type__in=attachment_type)
 
-    def get_input_message_metadata(self, resource_file_mapping: dict[str, list[str]]):
-        metadata = {"openai_thread_checkpoint": True, **self.get_trace_metadata()}
+    def get_input_message_metadata(self, resource_file_mapping: dict[str, list[str]]) -> dict:
         file_ids = set([file_id for file_ids in resource_file_mapping.values() for file_id in file_ids])
-        metadata["openai_file_ids"] = list(file_ids)
-        return metadata
+        return self.get_output_message_metadata(list(file_ids))
```

Comment:
Nit: Returning output message metadata for input message metadata is confusing. How about renaming  `get_output_message_metadata` to something like `get_openai_metadata`?